laurent22/joplin
# Desktop: Resolves #8749: Use plain text editor in safe mode

# Summary

This pull request enables the plain text editor when in safe mode.

This should resolve #8749.

# Testing

1. Enable safe mode
2. Open a note
3. Edit the just-opened note
4. Switch to a different note
5. Switch back to the note opened in step 3

In step 5, edits should be preserved.
